#51 Implement presentation creation & return resource

kayoumido opened the pull request.

kayoumido committed:
#51 Implement presentation creation & return resource

fabio10000 commented on src/app/Http/Controllers/API/PresentationController.php:
```diff
@@ -38,7 +39,27 @@ public function index(User $user)
+
+        // data is valid
+        // create the new presentation
+        $presentation = Presentation::create($request->only([
+            'title',
+            'date'
+        ]));
+
+        // link the new presentation w/ it's presenter
+        // i.e the current user
+        $presentation->users()->attach($user, [
+            'role' => 'presenter'
+        ]);
```
Here you may do something like `$user->presentations()->create($request->..., ['role' => 'presenter'])` not shure about the syntax concerning the pivot param but it will be better than create a presentation without any defined presenter and then assign it

kayoumido replied:
`ba62ba3 `

kayoumido committed:
#51 Create presentaion via the user

This is to make sure we don't have a presentation w/o a presenter even tho it was brief
